### DISCUSSION TITLE
Where are clipboard items stored?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [hluk/CopyQ#1656] [Q&A] [answered]
When using CopyQ to only store ephemeral clipboard data (no clips stored once CopyQ is closed), where is CopyQ storing the clipboard history data that is displayed in the main UI?  Is it simply stored in RAM or is it also written to storage?

**Comment by hluk** [comment, marked as answer]
If you **disable "Save Items"** in Tabs configuration (for a specific tab), items are stored **only in the main memory** (RAM) and not written to disk.

**Reply by Gitoffthelawn** [reply]
@hluk Great! Thank you!  That's exactly what I wanted to verify.

BTW, I noticed a tiny, tiny bug on the tab configurations page that, at least for me, the word "default" does not quite fit in the dropdown menu.  I can file an issue report and then one of us can fix it, or if you prefer, you can fix it without any issue report... whatever works best for you. :)

**Reply by hluk** [reply]
> the word "default" does not quite fit in the dropdown menu

I think that could be a Qt or system theme issue. Worked for me using all the styles Qt has available on my system.

**Reply by Gitoffthelawn** [reply]
Ok... it's obviously a very minor issue.  Do you specify the width of the dropdown field, or is Qt doing it on its own?

I love cross-platform software and libraries, but I get the sense that projects like Qt and GTK don't perform sufficient testing outside of Linux.

**Reply by hluk** [reply]
> Do you specify the width of the dropdown field, or is Qt doing it on its own?

This is handled by Qt.

> I love cross-platform software and libraries, but I get the sense that projects like Qt and GTK don't perform sufficient testing outside of Linux.

Testing UI is hard. It's either a manual job or an automated screenshot comparison (which could break often).

**Reply by Gitoffthelawn** [reply]
> This is handled by Qt.  

Thanks.  Good to know.

> Testing UI is hard. It's either a manual job or an automated screenshot comparison (which could break often).

Definitely.  And then there are the variables of DPI and i18n.
